GA/config.py

```
@@ -0,0 +1,164 @@
+
+
+# Controls for the optimisation vector - BLADES
+OPTIMIZE_STAGE = [False, False, False]
+ROTATING = [True, False, False]
+NUM_RADIALSECTIONS = 2  # Define the number of radial sections at which the blade profiles for each stage will be defined. 
+NUM_STAGES = 3  # Define the number of stages (i.e. total count of rotors + stators)
+REFERENCE_SECTION_ANGLES = [np.deg2rad(19), 0, 0]  # Reference angles at the reference section (typically 75% of blade span)
+BLADE_DIAMETERS = [2.1336, 2.2098, 2.2098]
+tipGap = 0.01016  # 1.016 cm tip gap
+
+with pushd(parent_dir):
```

Ruff (0.8.2)

Undefined name `parent_dir`

(F821)